apps/homework/templatetags/homework_tags.py

```diff
@@ -48,7 +48,21 @@ def get_last_grade_teacher(submission):
 def get_item(dictionary, key):
     return dictionary.get(key)
 
+def previous(some_list, current_index):
+    try:
+        return some_list[int(current_index) - 1]
+    except:
+        return ''
+
+def next(some_list, current_index):
+    try:
+        return some_list[int(current_index) + 1]
+    except:
```

Comment:
It probably does make sense to use a general try/except, but what happens if we only catch IndexOutOfBounds type exceptions?

Reply:
Catching `IndexError` exceptions only. Probably easier to intuit now.